sopel/modules/wikipedia.py

```diff
@@ -189,7 +189,11 @@ def say_snippet(bot, trigger, server, query, show_url=True):
         snippet = re.sub(r"\s+", " ", snippet)
     except KeyError:
         if show_url:
-            bot.reply("Error fetching snippet for \"{}\".".format(page_name))
+            msg = 'Error fetching snippet for "{}".'.format(page_name)
+            if commanded:
+                bot.reply(msg)
+            else:
+                bot.say(msg)
         return
```

Comment:
This logic doesn't entirely make sense. While there's no code path to call `say_snippet()` with `..., show_url=False, commanded=True` _right now_, there might be in the future.

Given the addition of `commanded` as an explicit parameter, there's no reason any more to abuse the `show_url` argument's value for inferring whether to output an error or stay silent.